backend/app/api/routes/documents.py

```
@@ -1,17 +1,26 @@
+from fastapi.responses import JSONResponse
+from fastapi import HTTPException
 
 from app.crud import DocumentCrud, CollectionCrud
 from app.models import Document
 from app.utils import APIResponse, load_description, get_openai_client
 from app.api.deps import CurrentUser, SessionDep, CurrentUserOrgProject
 from app.core.cloud import AmazonCloudStorage
 from app.crud.rag import OpenAIAssistantCrud
+from app.core.doctransform import service as transformation_service
+from app.core.doctransform.registry import (
+    get_file_format, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)